user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/EditBatchDialog.java

```diff
@@ -137,9 +137,24 @@ private void onBatchNameChanged(
   }
 
   private void onRemoveLastRowClicked(ClickEvent<Button> clickEvent) {
+    SampleInfo sampleInfo = spreadsheet.getLastRowData();
+    fireEvent(new RemoveRowEvent(this, clickEvent.isFromClient(),
+        sampleInfo));
```

Comment:
no row removal? Sorry I do not get this change.

Reply:
This transfers the event to the outside where it is checked if it can be removed - I can rename it to "RemoveRowClickedEvent" or something?

Reply:
I renamed it and added the running number of the sample (# column) as a parameter in order to provide the user with that info.